<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestClientClass</class>
 <widget class="QMainWindow" name="QuestClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QuestClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0" rowspan="2">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0" colspan="2">
       <widget class="QWidget" name="modelControllWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>61</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string>方案</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="solution1Choice">
             <property name="text">
              <string>方案1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" colspan="3">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>348</width>
               <height>18</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>13</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QRadioButton" name="littleChoice">
             <property name="text">
              <string>方案2-小抓斗方案</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QRadioButton" name="bigChoice">
             <property name="text">
              <string>方案2-大抓斗方案</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QRadioButton" name="bigChoiceDump">
             <property name="text">
              <string>方案2-大抓斗高峰期方案</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_19">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>速度</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="simInterval">
             <property name="text">
              <string>0.5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="animationMode">
             <property name="text">
              <string>动画</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="startSim">
             <property name="text">
              <string>开始</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="resetSim">
             <property name="text">
              <string>重置</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="continueSim">
             <property name="text">
              <string>继续</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QWidget" name="agvSpeedWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>160</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_1">
               <property name="text">
                <string>垃圾吊1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="agvSpeed1">
               <property name="text">
                <string>600</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>mm/s</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>升降1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="hoistSpeed1">
               <property name="text">
                <string>600</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_26">
               <property name="text">
                <string>mm/s</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>垃圾吊2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="agvSpeed2">
               <property name="text">
                <string>600</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>mm/s</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>升降2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="hoistSpeed2">
               <property name="text">
                <string>600</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_27">
               <property name="text">
                <string>mm/s</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>垃圾吊3</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="agvSpeed3">
               <property name="text">
                <string>600</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>mm/s</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>升降3</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="hoistSpeed3">
               <property name="text">
                <string>600</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_28">
               <property name="text">
                <string>mm/s</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>垃圾吊V4</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="agvSpeed4">
               <property name="text">
                <string>600</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_25">
               <property name="text">
                <string>mm/s</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>升降4</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="hoistSpeed4">
               <property name="text">
                <string>600</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_29">
               <property name="text">
                <string>mm/s</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="sendAGVSpeed">
           <property name="text">
            <string>应用</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QWidget" name="levelWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>221</width>
          <height>160</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>2号料位 最高</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="levelH2">
               <property name="text">
                <string>60</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>最低</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="levelL3">
               <property name="text">
                <string>30</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>最低</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="levelL2">
               <property name="text">
                <string>30</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>1号料位 最高</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="levelH1">
               <property name="text">
                <string>60</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>最低</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="levelL1">
               <property name="text">
                <string>30</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>3号料位 最高</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="levelH3">
               <property name="text">
                <string>60</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="sendLevel">
           <property name="text">
            <string>应用</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QWidget" name="timeWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_21">
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>抓料时间</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="loadTime">
             <property name="text">
              <string>90</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>拌料时间</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="unloadDLTime">
             <property name="text">
              <string>60</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_23">
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>投料时间</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="unloadTime">
             <property name="text">
              <string>90</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="sendTime">
           <property name="text">
            <string>应用</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="reportWidget" native="true">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="updateReport">
         <property name="text">
          <string>更新报表</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="exportReport">
         <property name="text">
          <string>导出报表</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>797</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTextBrowser" name="reportBrowser">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QuestClient.qrc"/>
 </resources>
 <connections/>
</ui>
